co-simulation tool/README.md

```diff
@@ -0,0 +1,418 @@
+- Add the **bin** directory of the created directory **apache-maven-3.8.1** to the **PATH** environment variable.
+
+  **Windows**
+
+  Add the path to bin directory to the Path in the System variables.
+
+  **Linux (Ubuntu)**
+
+  ```
+  export PATH=path_to_apache_maven-3.8.1/apache-maven-3.8.1/bin:$PATH
+  ```
+
+- Verify the installation by running **mvn -v** in a new terminal. The output should look something like the following. You can check the Java version and maven path.
+
+  ```
+  Maven home: C:\Program Files\apache-maven-3.8.1\bin\..
+  Java version: 16, vendor: Oracle Corporation, runtime: C:\Program Files\Java\jdk-16
+  Default locale: en_US, platform encoding: Cp1252
+  OS name: "windows 10", version: "10.0", arch: "amd64", family: "windows"
+  ```
+
+#### Installation of SUMO-1.8.0
+
+**Windows**
+
+- Download the binaries from this [link](https://sourceforge.net/projects/sumo/files/sumo/version%201.8.0/).
+
+- Open the installer and follow the instruction to install it. Select **set SUMO_HOME.**
+
+- Verify the installation by running sumo-gui in a terminal.
+
+  ```
+  sumo-gui
+  ```
+
+**Linux (Ubuntu)**
+
+- Install the required tools and libraries
+
+  ```
+  sudo apt-get install git cmake python3 g++ libxerces-c-dev libfox-1.6-dev libgdal-dev libproj-dev libgl2ps-dev
+  ```
+
+- Download the source code SUMO-1.8.0 from this [link](https://github.com/eclipse/sumo/releases/tag/v1_8_0).
+
+- Extract the source code to a folder, for instance, folder *sumo*
+
+- Go to the location of the folder of source code and set the environment variable **SUMO_HOME**
+
+  ```
+  export SUMO_HOME="$PWD/sumo"
+  ```
+
+- Build the SUMO binaries
+
+  ```
+   mkdir sumo/build/cmake-build && cd sumo/build/cmake-build
+   cmake ../..
+   make -j$(nproc)
+  ```
+
+- Installing the SUMO binaries
+
+  ```
+  sudo make install
+  ```
+
+- Adjust **SUMO_HOME** variable to the install directory  (usually /usr/local/share/sumo)
+
+  ```
+  export SUMO_HOME=/usr/local/share/sumo
+  ```
+
+- Verify the installation by running sumo-gui in a terminal.
+
+  ```
+  sumo-gui
+  ```
+
+## Build Co-simulation Tool
+
+- Download zip file of Co-simulation tool or clone the code from this [link](https://github.com/usdot-fhwa-stol/carma-simulation/tree/feature/mosaic-carla-sumo-ambassadors).
+
+  ```
+  git clone https://github.com/usdot-fhwa-stol/carma-simulation.git
+  ```
+
+- If you use git clone the code, you need to use the following command to go to the co-simulation tool branch.
+
+  ```
+  git checkout feature/mosaic-carla-sumo-ambassadors
+  ```
+
+- Go to co-simulation folder and build co-simulation tool using the following command
+
+  ```
+  cd 'co-simulation tool'
+  mvn clean install
+  ```
+
+- The above mvn command executes all tests as well. In order to skip test execution, run the following command:
+
+  ```
+  mvn clean install -DskipTests
+  ```
+
+- After building, go to bundle\target directory, a zip file of co-simulation bundle can be found.
```

Comment:
Please specify the name of "a zip file of co-simulation bundle".

Reply:
The version "21.1-SNAPSHOT" has been changed to "1.0". The name of bundle "eclipse-mosaic-21.1-SNAPSHOT" has been changed to "carla-sumo-mosaic-1.0". The bundle name was updated in the README.md.